kimchi/src/tests/not.rs

```diff
@@ -46,6 +42,36 @@ const RNG_SEED: [u8; 32] = [
     89, 29, 13, 250, 215, 172, 130, 24, 164, 162,
 ];
 
+// Creates as many negations as the number of inputs. The inputs must fit in the native field.
+// We start at the row 0 using generic gates to perform the negations.
+// Input: a vector of words to be negated, and the number of bits (all the same)
+// Panics if the bits length is too small for the inputs
```

Comment:
possible not to panic and error instead?

Reply:
Okay, I made the function inside the not polynomial to return an error, and then the function for tests is the one that panics